hdr_explainer.md

```diff
@@ -51,14 +53,14 @@ let hdrConfig = {
 
 navigator.mediaCapabilities.decodingInfo(hdrConfig).then(function(info) {
 	if (info.supported) {
-		// Playback is supported! See also, info.smooth and info.powerEfficient. 
+		// Playback is supported! See also, info.smooth and info.powerEfficient.
 	}
 });
 ```
 
 # Implementations
 
-As of 2020/11, here's the unofficial status of user agent implementations for the above:
+As of 2020/21, here's the unofficial status of user agent implementations for the above:
```

Comment:
This section needs to be updated per the following:

- https://caniuse.com/mdn-css_at-rules_media_dynamic-range
- https://caniuse.com/mdn-css_at-rules_media_video-dynamic-range
- https://chromestatus.com/feature/6640863931269120
-